xocto/localtime.py

```diff
@@ -664,3 +664,22 @@ def translate_english_month_to_spanish(month: int) -> str:
         "December": "deciembre",
     }
     return month_name_lookup[month_name]
+
+
+def get_month_datetime_ranges(
+    range: ranges.FiniteDatetimeRange,
```

Comment:
🤔 To match the other functions in this file, could this function be modified to take an optional `tz` argument, so that the timezone can be specified?

Alternatively, we should probably assert that the input range boundaries are localised in some way.